users/tests.py

```diff
@@ -1,3 +1,36 @@
-# from django.test import TestCase
+from datetime import datetime
+import pytest
+from django.contrib.auth import get_user_model
+from users.models import Profile
 
-# Create your tests here.
+
+@pytest.fixture
+def create_user():
```

Comment:
The user in the fixture is used like a predefined user 
so IMHO it would be better to call it something like user1
```suggestion
def user1():
```